### ISSUE TITLE
✔️ 🐞 Bug Report: Loop Builder - Post Heading widget does not show if "Hide Page Title" is enabled (ED-8557)

### ISSUE THREAD

**Issue by Nagotkom** [elementor/elementor#20207]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When using the loop builder and adding a post heading it will not show if I in the settings for the page enable hide title. 

### Steps to reproduce

1. Create a container on a page
2. Add the loop builder widget in the container 
3. Create a template with a Post Title widget
4. Save and exit template
5. Return to page click Settings in lower left corner. 
6. Switch on or off hide title 
7. Watch the Post title disappear and appear. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

= Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.5.17
	PHP Version: 7.4.32
	PHP Memory Limit: 256M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.3
	Site URL: https://nagotkommunikation.se
	Home URL: https://nagotkommunikation.se
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: sv-SE
	Timezone: Europe/Stockholm
	Admin Email: emil@nagotkommunikation.se
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: sv_SE
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.1 Safari/605.1.15

== Active Plugins ==
	Elementor
		Version: 3.8.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.8.0
		Author: Elementor.com

== Elementanvändning ==
	
	archive : 2
		heading : 2
		nav-menu : 2
		posts : 4
	footer : 1
		container : 3
		image : 2
		social-icons : 1
		text-editor : 3
	header : 1
		container : 2
		image : 2
		nav-menu : 1
		search-form : 1
	wp-page : 15
		accordion : 2
		button : 23
		container : 18
		form : 3
		heading : 36
		html : 6
		icon-box : 9
		icon-list : 3
		image : 20
		loop-grid : 1
		menu-anchor : 1
		portfolio : 2
		posts : 1
		sitemap : 1
		text-editor : 45
	single-post : 2
		button : 2
		container : 9
		divider : 2
		form : 1
		heading : 3
		image : 9
		loop-grid : 1
		post-info : 2
		posts : 1
		social-icons : 1
		table-of-contents : 1
		text-editor : 7
		theme-post-content : 1
		theme-post-excerpt : 1
		theme-post-featured-image : 2
		theme-post-title : 2
	search-results : 1
		archive-posts : 1
		heading : 1
		search-form : 1
	loop-item : 3
		container : 3
		theme-post-excerpt : 1
		theme-post-featured-image : 2
		theme-post-title : 2
	error-404 : 1
		heading : 3
		text-editor : 1

== Elementor Experiments ==
	Optimized DOM Output: Aktiv
	Improved Asset Loading: Aktiv
	Förbättrad CSS-laddning: Aktiv
	Inline Font Icons: Aktiv
	Tillgänglighetsförbättringar : Aktiv som standard
	Ytterligare anpassade brytpunkter: Inaktiv som standard
	Importera/exportera webbplats-kit: Aktiv som standard
	Dölj inbyggda WordPress-widgetar från sökresultat: Aktiv som standard
	admin_menu_rearrangement: Inaktiv som standard
	Flexbox Container: Aktiv
	Default to New Theme Builder: Aktiv som standard
	Hello Theme Header & Footer: Inaktiv som standard
	Landningssidor: Aktiv som standard
	Färgprovtagare: Aktiv som standard
	Favoritwidgetar: Aktiv som standard
	Admin Top Bar: Aktiv som standard
	Sidövergångar: Aktiv som standard
	Notes: Aktiv som standard
	Loop: Aktiv
	Formulärinlämningar: Aktiv som standard
	Scroll Snap: Inaktiv

== Logg ==
	
Log: showing 20 of 332022-09-22 08:44:45 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-09-22 08:44:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.6',
  'to' => '3.7.7',
)]
2022-09-22 08:44:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.6',
  'to' => '3.7.7',
)]
2022-10-03 08:40:16 [info] elementor::elementor_updater Started 
2022-10-03 08:40:16 [info] Elementor/Upgrades - _on_each_version Start  
2022-10-03 08:40:17 [info] Elementor/Upgrades - _on_each_version Finished 
2022-10-03 08:40:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.7',
  'to' => '3.7.8',
)]
2022-10-03 08:40:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.7',
  'to' => '3.7.8',
)]
2022-10-30 20:44:27 [info] elementor::elementor_updater Started 
2022-10-30 20:44:27 [info] Elementor/Upgrades - _on_each_version Start  
2022-10-30 20:44:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.8.0',
)]
2022-10-30 20:44:28 [info] Elementor/Upgrades - _on_each_version Finished 
2022-10-30 20:44:28 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-10-30 20:44:28 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-10-30 20:44:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.8.0',
)]
2022-10-30 20:45:24 [info] elementor-pro::elementor_pro_updater Started 
2022-10-30 20:45:24 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-10-30 20:45:24 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-10-30 20:45:24 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]
2022-10-30 20:45:24 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.7',
  'to' => '3.8.0',
)]

JS: showing 2 of 2JS: 2022-09-16 13:51:32 [error X 1][https://nagotkommunikation.se/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.6:3:287002] undefined is not an object (evaluating \'T.params.args.data.id\') 
JS: 2022-10-30 19:59:41 [error X 4][https://nagotkommunikation.se/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.0:3:1061529] null is not an object (evaluating \'$.getBoundingClientRect\') 

PHP: showing 7 of 7PHP: 2022-09-18 11:26:14 [error X 2][/home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::294] Uncaught Error: Call to a member function update_meta() on null in /home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:294
Stack trace:
#0 /home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(5, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/nagotkom/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/nagotkom/public_html/wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home/nagotkom/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)
#5 /home/nagotkom/publi [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-30 20:59:33 [error X 2][/home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::294] Uncaught Error: Call to a member function update_meta() on bool in /home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:294
Stack trace:
#0 /home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/nagotkom/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/nagotkom/public_html/wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home/nagotkom/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)
#5 /home/nagotkom/pu [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-30 21:00:12 [notice X 5][/home/nagotkom/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::167] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-30 21:14:36 [notice X 18][/home/nagotkom/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-10-30 21:14:37 [notice X 17][/home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/nagotkom/public_html/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2022-10-31 18:53:58 [notice X 1][/home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/nagotkom/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(84): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/nagotkom/public_html/wp-content/plugins/elementor/data/base/endpoint.php(166): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> update_item()
#3: /home/nagotkom/public_html/wp-content/plugins/elementor/data/base/endpoint.php(307): Elementor\Data\Base\Endpoint -> base_callback()
#4: /home/nagotkom/public_html/wp-includes/rest-api/class-wp-rest-server.php(1143): Elementor\Data\Base\Endpoint -> Elementor\Data\Base\{closure}()
',
)]
PHP: 2022-10-31 18:56:11 [error X 1][/home/nagotkom/public_html/wp-content/plugins/elementor/includes/managers/widgets.php::408] Uncaught Error: Call to a member function is_editable_by_current_user() on bool in /home/nagotkom/public_html/wp-content/plugins/elementor/includes/managers/widgets.php:408
Stack trace:
#0 [internal function]: Elementor\Widgets_Manager->ajax_render_widget(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /home/nagotkom/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/nagotkom/public_html/wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /home/nagotkom/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)
#4 /home/nagotkom/public_html/wp-includes/plugin.php(476): WP_Hook->do_action(Array)
#5 /home/nagotkom/public_html/wp-admin/admin-ajax.php(187): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Inkompatibel

== Elementor Pro - Compatibility Tag ==

**Comment by Jeanseb123** [comment]
same here

**Comment by fwunder** [comment]
Same problem here. Obvious fix is custom css on post. Just kind of a pain for lots of posts.

```
.entry-title {
display: none;
}​
```

**Comment by nicholaszein** [comment]
Hello there @Nagotkom, @drau1, @Jeanseb123, and @fwunder!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.8.1**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
